toolbox

```diff
@@ -872,6 +872,10 @@ create()
             --volume /dev:/dev:rslave \
             --volume /media:/media:rslave \
             --volume /mnt:/mnt:rslave \
+            --volume /run:/run/host/run:rslave \
```

Comment:
Hm, but I see you chose `/run/host` instead of `/host`.  Standardizing that path as much as we can is useful because other tools may come to rely on it.  For example, things like Systemtap may want to look at `/host/usr/lib/modules` to find the host kernel.

I chose `/host` because `oc debug node` from OpenShift did.

Reply:
Interesting. I didn't know OpenShift uses `/host`. I did notice that you had went with `/host`, not `/run/host`.

I went with `/run/host` for no other reason than that we already had it [for resolv.conf](https://github.com/debarshiray/toolbox/commit/04732139074b60dabe3087c37afbd99bdbf14a8a), and the original reason was just that it's the location that Flatpak uses.

I am happy to switch to `/host` or add some symbolic links or whatever.

Reply:
Let's merge this, and think of `/run/host` versus `/host` in a separate PR or issue.